{
  "Faults": [
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T07:19:00.000+0000",
      "latitude": 51.31376722209966,
      "estimatedRestoration": "2022-02-17T17:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM6092",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.31434,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.74795
            },
            {
              "py/object": "Point",
              "latitude": 51.31377,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.74761
            },
            {
              "py/object": "Point",
              "latitude": 51.31325,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.74599
            },
            {
              "py/object": "Point",
              "latitude": 51.31379,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.74609
            },
            {
              "py/object": "Point",
              "latitude": 51.31434,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.74795
            }
          ],
          "subCodes": [
            "GU16 7HE",
            "GU16 7HF",
            "GU16 7HQ",
            "GU16 7JA"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM6092",
      "engineerOnSiteTime": "2022-02-17T08:30:00.000+0000",
      "name": "GU16 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "GU16 7HE",
        "GU16 7HF",
        "GU16 7HQ",
        "GU16 7JA"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -0.74795,
              51.31434
            ],
            [
              -0.74761,
              51.31377
            ],
            [
              -0.74599,
              51.31325
            ],
            [
              -0.74609,
              51.31379
            ],
            [
              -0.74795,
              51.31434
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-02-17T08:30:00.000+0000",
      "UUID": "0560578d-dff7-43df-ab52-ab8bbd09c297",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7469107672291211,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T08:00:00.000+0000",
      "latitude": 51.58050315370718,
      "estimatedRestoration": "2022-02-17T17:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205357'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205357",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.58128,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.87039
            },
            {
              "py/object": "Point",
              "latitude": 51.58801,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.86968
            },
            {
              "py/object": "Point",
              "latitude": 51.57331,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.86856
            },
            {
              "py/object": "Point",
              "latitude": 51.57893,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.87021
            },
            {
              "py/object": "Point",
              "latitude": 51.58128,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.87039
            }
          ],
          "subCodes": [
            "RG9 6SD",
            "RG9 6SE",
            "RG9 6SL",
            "RG9 6SN",
            "RG9 6SP"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205357",
      "name": "RG9 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG9 6SD",
        "RG9 6SE",
        "RG9 6SL",
        "RG9 6SN",
        "RG9 6SP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -0.87039,
              51.58128
            ],
            [
              -0.86968,
              51.58801
            ],
            [
              -0.86856,
              51.57331
            ],
            [
              -0.87021,
              51.57893
            ],
            [
              -0.87039,
              51.58128
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "b5c95241-a75c-4de2-b59e-65f7318c8ff8",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8696673122205847,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T08:00:00.000+0000",
      "latitude": 51.60922020710671,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205504'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205504",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.60921,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.60525
            }
          ],
          "name": "HP9 2UP",
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205504",
      "name": "HP9 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP9 2UP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.6052381329574872,
          51.60922020710671
        ],
        "type": "Point"
      },
      "UUID": "b0fa59b2-ba76-414f-9b30-1f649d9fbc03",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6052381329574872,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T08:30:00.000+0000",
      "latitude": 51.58944142220886,
      "estimatedRestoration": "2022-02-17T16:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205352'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205352",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.59262,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.87694
            },
            {
              "py/object": "Point",
              "latitude": 51.58771,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.87666
            },
            {
              "py/object": "Point",
              "latitude": 51.58801,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.86968
            },
            {
              "py/object": "Point",
              "latitude": 51.59262,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.87694
            }
          ],
          "subCodes": [
            "RG9 6SL",
            "RG9 6SX",
            "RG9 6TA"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205352",
      "name": "RG9 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG9 6SL",
        "RG9 6SX",
        "RG9 6TA"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -0.87694,
              51.59262
            ],
            [
              -0.87666,
              51.58771
            ],
            [
              -0.86968,
              51.58801
            ],
            [
              -0.87694,
              51.59262
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "4046b756-fba1-4765-b59a-8ee061b34e1a",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8743967598146679,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T07:19:00.000+0000",
      "latitude": 51.60030309021499,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM6086",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.60034,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.2581
            },
            {
              "py/object": "Point",
              "latitude": 51.60030309021499,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.257451051997132
            },
            {
              "py/object": "Point",
              "latitude": 51.59994,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.2569
            },
            {
              "py/object": "Point",
              "latitude": 51.60034,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.2581
            }
          ],
          "subCodes": [
            "OX11 0AX",
            "OX11 8QR"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM6086",
      "engineerOnSiteTime": "2022-02-17T08:20:00.000+0000",
      "name": "OX11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX11 0AX",
        "OX11 8QR"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.2581,
              51.60034
            ],
            [
              -1.257451051997132,
              51.60030309021499
            ],
            [
              -1.2569,
              51.59994
            ],
            [
              -1.2581,
              51.60034
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-02-17T08:30:00.000+0000",
      "UUID": "2526e77c-c9e7-469d-bb50-c77943fb6734",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.257451051997132,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:00:00.000+0000",
      "latitude": 51.192832467541976,
      "estimatedRestoration": "2022-02-17T14:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '204633'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/204633",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.19546,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.55847
            },
            {
              "py/object": "Point",
              "latitude": 51.1988,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.55682
            },
            {
              "py/object": "Point",
              "latitude": 51.19895,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.53999
            },
            {
              "py/object": "Point",
              "latitude": 51.19379,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.52088
            },
            {
              "py/object": "Point",
              "latitude": 51.18854,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.52833
            },
            {
              "py/object": "Point",
              "latitude": 51.18556,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.54384
            },
            {
              "py/object": "Point",
              "latitude": 51.19546,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.55847
            }
          ],
          "subCodes": [
            "SP11 7AZ",
            "SP11 7BA",
            "SP11 7BB",
            "SP11 7BD",
            "SP11 7BE",
            "SP11 7BG",
            "SP11 7BH",
            "SP11 7BJ",
            "SP11 7BQ",
            "SP11 7DB",
            "SP11 7DP",
            "SP11 7DR",
            "SP11 7DS",
            "SP11 7DT",
            "SP11 7DU",
            "SP11 8AL",
            "SP11 8AN",
            "SP11 8AQ"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "204633",
      "name": "SP11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP11 7AZ",
        "SP11 7BA",
        "SP11 7BB",
        "SP11 7BD",
        "SP11 7BE",
        "SP11 7BG",
        "SP11 7BH",
        "SP11 7BJ",
        "SP11 7BQ",
        "SP11 7DB",
        "SP11 7DP",
        "SP11 7DR",
        "SP11 7DS",
        "SP11 7DT",
        "SP11 7DU",
        "SP11 8AL",
        "SP11 8AN",
        "SP11 8AQ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.55847,
              51.19546
            ],
            [
              -1.55682,
              51.1988
            ],
            [
              -1.53999,
              51.19895
            ],
            [
              -1.52088,
              51.19379
            ],
            [
              -1.52833,
              51.18854
            ],
            [
              -1.54384,
              51.18556
            ],
            [
              -1.55847,
              51.19546
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "bb908877-ba77-4bd9-9576-da4f7f656edb",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.5370295185818783,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:00:00.000+0000",
      "latitude": 51.71054687873916,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205408'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205408",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.71078,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.22556
            }
          ],
          "name": "OX4 4YP",
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205408",
      "name": "OX4 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX4 4YP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.2254567005793777,
          51.71054687873916
        ],
        "type": "Point"
      },
      "UUID": "1900226b-335d-4f01-b9e2-f46c9288e47e",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2254567005793777,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:00:00.000+0000",
      "latitude": 51.40985728748771,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205948'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205948",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.40953,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.34615
            },
            {
              "py/object": "Point",
              "latitude": 51.4108,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.34544
            },
            {
              "py/object": "Point",
              "latitude": 51.41047,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.34389
            },
            {
              "py/object": "Point",
              "latitude": 51.4082,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.3404
            },
            {
              "py/object": "Point",
              "latitude": 51.40953,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.34615
            }
          ],
          "subCodes": [
            "RG14 1QT",
            "RG14 1QX",
            "RG14 1QY",
            "RG14 1RA",
            "RG14 1RB",
            "RG14 1RD",
            "RG14 1RJ"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205948",
      "name": "RG14 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG14 1QT",
        "RG14 1QX",
        "RG14 1QY",
        "RG14 1RA",
        "RG14 1RB",
        "RG14 1RD",
        "RG14 1RJ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.34615,
              51.40953
            ],
            [
              -1.34544,
              51.4108
            ],
            [
              -1.34389,
              51.41047
            ],
            [
              -1.3404,
              51.4082
            ],
            [
              -1.34615,
              51.40953
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "f8432685-c8bf-4f16-a098-faa45767fc26",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3443630602299954,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:00:00.000+0000",
      "latitude": 51.42143523633786,
      "estimatedRestoration": "2022-02-17T17:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205367'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205367",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.4221,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.8326
            },
            {
              "py/object": "Point",
              "latitude": 51.42143523633786,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.8318582245217157
            },
            {
              "py/object": "Point",
              "latitude": 51.42073,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.83107
            }
          ],
          "subCodes": [
            "RG40 5QB",
            "RG40 5QE"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205367",
      "name": "RG40 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG40 5QB",
        "RG40 5QE"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -0.8326,
              51.4221
            ],
            [
              -0.8318582245217157,
              51.42143523633786
            ],
            [
              -0.83107,
              51.42073
            ],
            [
              -0.8326,
              51.4221
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "244b554a-dfa6-4f77-b6b7-5d105bb2a2d8",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8318582245217157,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:00:00.000+0000",
      "latitude": 51.234352204231335,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205154'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205154",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.23282,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.23982
            },
            {
              "py/object": "Point",
              "latitude": 51.23429,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.23825
            },
            {
              "py/object": "Point",
              "latitude": 51.23876,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.23256
            },
            {
              "py/object": "Point",
              "latitude": 51.23145,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.23007
            },
            {
              "py/object": "Point",
              "latitude": 51.23081,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.23468
            },
            {
              "py/object": "Point",
              "latitude": 51.23282,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.23982
            }
          ],
          "subCodes": [
            "BA13 4AA",
            "BA13 4AB",
            "BA13 4AD",
            "BA13 4AJ",
            "BA13 4AQ",
            "BA13 4GF"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205154",
      "name": "BA13 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BA13 4AA",
        "BA13 4AB",
        "BA13 4AD",
        "BA13 4AJ",
        "BA13 4AQ",
        "BA13 4GF"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -2.23982,
              51.23282
            ],
            [
              -2.23825,
              51.23429
            ],
            [
              -2.23256,
              51.23876
            ],
            [
              -2.23007,
              51.23145
            ],
            [
              -2.23468,
              51.23081
            ],
            [
              -2.23982,
              51.23282
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "d04c8785-bc80-4532-8169-32b3c65feb5b",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.235521697401507,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:00:00.000+0000",
      "latitude": 51.641899913696925,
      "estimatedRestoration": "2022-02-17T15:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205387'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205387",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.64262,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.93997
            },
            {
              "py/object": "Point",
              "latitude": 51.641899913696925,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.9386219874616464
            },
            {
              "py/object": "Point",
              "latitude": 51.64113,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.93713
            }
          ],
          "subCodes": [
            "SN6 6PA",
            "SN6 6PW"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205387",
      "name": "SN6 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN6 6PA",
        "SN6 6PW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.93997,
              51.64262
            ],
            [
              -1.9386219874616464,
              51.641899913696925
            ],
            [
              -1.93713,
              51.64113
            ],
            [
              -1.93997,
              51.64262
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "4315302c-4ff0-44ca-844b-b48002133fb7",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9386219874616464,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:00:00.000+0000",
      "latitude": 50.90061510077625,
      "estimatedRestoration": "2022-02-17T16:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205386'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205386",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 50.90062,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.28495
            }
          ],
          "name": "DT11 0RP",
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205386",
      "name": "DT11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "DT11 0RP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.2850399850650835,
          50.90061510077625
        ],
        "type": "Point"
      },
      "UUID": "50119453-2922-4571-88ca-2b94633a4e26",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.2850399850650835,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:00:00.000+0000",
      "latitude": 50.724362819077015,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205728'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205728",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 50.72433,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.08649
            }
          ],
          "name": "BH16 6LA",
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205728",
      "name": "BH16 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH16 6LA"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.0864942859648608,
          50.724362819077015
        ],
        "type": "Point"
      },
      "UUID": "faef4aa4-01d3-4c44-9bcb-3439b5723e5f",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.0864942859648608,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:00:00.000+0000",
      "latitude": 50.95446565713589,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205185'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205185",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 50.95445,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.63705
            }
          ],
          "name": "BA21 4AW",
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205185",
      "name": "BA21 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BA21 4AW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.637022015869029,
          50.95446565713589
        ],
        "type": "Point"
      },
      "UUID": "f124968d-b7d1-4579-b366-b72e5b91c4eb",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.637022015869029,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T10:40:00.000+0000",
      "latitude": 51.45464024309513,
      "estimatedRestoration": "2022-02-17T15:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM4927",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.45452,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.05429
            },
            {
              "py/object": "Point",
              "latitude": 51.45424,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.05401
            },
            {
              "py/object": "Point",
              "latitude": 51.45423,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.05034
            },
            {
              "py/object": "Point",
              "latitude": 51.45486,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.05024
            },
            {
              "py/object": "Point",
              "latitude": 51.4552,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.05084
            },
            {
              "py/object": "Point",
              "latitude": 51.45452,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.05429
            }
          ],
          "subCodes": [
            "RG31 5HJ",
            "RG31 5HL",
            "RG31 5HP",
            "RG31 5HR",
            "RG31 5HW",
            "RG31 5JT"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM4927",
      "engineerOnSiteTime": "2022-02-17T08:30:00.000+0000",
      "name": "RG31 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG31 5HJ",
        "RG31 5HL",
        "RG31 5HP",
        "RG31 5HR",
        "RG31 5HW",
        "RG31 5JT"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.05429,
              51.45452
            ],
            [
              -1.05401,
              51.45424
            ],
            [
              -1.05034,
              51.45423
            ],
            [
              -1.05024,
              51.45486
            ],
            [
              -1.05084,
              51.4552
            ],
            [
              -1.05429,
              51.45452
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "634a589c-e3d2-4c16-a9fb-0cdc3c9fc1da",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0517907761842218,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T10:40:00.000+0000",
      "latitude": 51.339935094489135,
      "estimatedRestoration": "2022-02-17T17:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM4987",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.33991,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.71458
            }
          ],
          "name": "GU15 1LZ",
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM4987",
      "engineerOnSiteTime": "2022-02-17T09:45:00.000+0000",
      "name": "GU15 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "GU15 1LZ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.7145983586756001,
          51.339935094489135
        ],
        "type": "Point"
      },
      "UUID": "4b9a6e91-4666-444d-b5ef-6c4064d32176",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7145983586756001,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-12T10:00:00.000+0000",
      "latitude": 51.52771493272998,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM2774",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.52771493272998,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.2513921496173496
            },
            {
              "py/object": "Point",
              "latitude": 51.52818,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.25046
            }
          ],
          "subCodes": [
            "NW10 6EG",
            "NW10 6LJ",
            "NW10 6LL"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM2774",
      "engineerOnSiteTime": "2022-02-13T09:00:00.000+0000",
      "name": "NW10 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "NW10 6EG",
        "NW10 6LJ",
        "NW10 6LL"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.2513921496173496,
          51.52771493272998
        ],
        "type": "Point"
      },
      "UUID": "f0b67263-5268-4a77-974e-53a58abfc43e",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.2513921496173496,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T11:33:00.000+0000",
      "latitude": 51.08884469383602,
      "estimatedRestoration": "2022-02-17T15:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM6317",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.0888,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.74684
            },
            {
              "py/object": "Point",
              "latitude": 51.09019,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.74265
            },
            {
              "py/object": "Point",
              "latitude": 51.08724,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.74194
            },
            {
              "py/object": "Point",
              "latitude": 51.0888,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.74684
            }
          ],
          "subCodes": [
            "GU27 1LS",
            "GU27 1LT",
            "GU27 1NJ",
            "GU27 1RD"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM6317",
      "name": "GU27 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "GU27 1LS",
        "GU27 1LT",
        "GU27 1NJ",
        "GU27 1RD"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -0.74684,
              51.0888
            ],
            [
              -0.74265,
              51.09019
            ],
            [
              -0.74194,
              51.08724
            ],
            [
              -0.74684,
              51.0888
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-02-17T12:40:00.000+0000",
      "UUID": "ad9d65fe-4337-4b09-8ab9-6113b3eb3fe1",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7439309197119566,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T11:52:00.000+0000",
      "latitude": 51.61951599922418,
      "estimatedRestoration": "2022-02-17T15:30:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM6333",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.61936,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.73789
            },
            {
              "py/object": "Point",
              "latitude": 51.61987,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.73594
            },
            {
              "py/object": "Point",
              "latitude": 51.61932,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.73498
            },
            {
              "py/object": "Point",
              "latitude": 51.61936,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.73789
            }
          ],
          "subCodes": [
            "HP11 1DN",
            "HP11 1DT",
            "HP11 1DU"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM6333",
      "name": "HP11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP11 1DN",
        "HP11 1DT",
        "HP11 1DU"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -0.73789,
              51.61936
            ],
            [
              -0.73594,
              51.61987
            ],
            [
              -0.73498,
              51.61932
            ],
            [
              -0.73789,
              51.61936
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-02-17T13:00:00.000+0000",
      "UUID": "fea3e1d3-560b-49bb-8397-baf63088e6ed",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7362840320627938,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T12:35:00.000+0000",
      "latitude": 51.39656750719157,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM6209",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.39526,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.48652
            },
            {
              "py/object": "Point",
              "latitude": 51.39656750719157,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.478689399671865
            },
            {
              "py/object": "Point",
              "latitude": 51.39925,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.4645
            },
            {
              "py/object": "Point",
              "latitude": 51.39473,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.48397
            },
            {
              "py/object": "Point",
              "latitude": 51.39526,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.48652
            }
          ],
          "subCodes": [
            "RG17 9SG",
            "RG17 9SJ",
            "RG17 9SQ"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM6209",
      "engineerOnSiteTime": "2022-02-17T10:30:00.000+0000",
      "name": "RG17 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG17 9SG",
        "RG17 9SJ",
        "RG17 9SQ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.48652,
              51.39526
            ],
            [
              -1.478689399671865,
              51.39656750719157
            ],
            [
              -1.4645,
              51.39925
            ],
            [
              -1.48397,
              51.39473
            ],
            [
              -1.48652,
              51.39526
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "e3544b56-36ac-4def-aef4-d45336fe2798",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.478689399671865,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T09:42:00.000+0000",
      "latitude": 50.58802541732326,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM6214",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 50.58791,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.47208
            }
          ],
          "name": "DT4 9BH",
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM6214",
      "engineerOnSiteTime": "2022-02-17T10:40:00.000+0000",
      "name": "DT4 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "DT4 9BH",
        "UM5"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.471970355084167,
          50.58802541732326
        ],
        "type": "Point"
      },
      "UUID": "33cb5be2-d96a-4ab2-bc10-08d21e3e183c",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.471970355084167,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T13:12:00.000+0000",
      "latitude": 51.68116547183463,
      "estimatedRestoration": "2022-02-17T18:30:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM6402",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.68116547183463,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.4567277958160496
            },
            {
              "py/object": "Point",
              "latitude": 51.69227,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.44652
            },
            {
              "py/object": "Point",
              "latitude": 51.69251,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.44225
            },
            {
              "py/object": "Point",
              "latitude": 51.69266,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.43785
            },
            {
              "py/object": "Point",
              "latitude": 51.69217,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.43614
            },
            {
              "py/object": "Point",
              "latitude": 51.69141,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.43278
            },
            {
              "py/object": "Point",
              "latitude": 51.69255,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.42472
            },
            {
              "py/object": "Point",
              "latitude": 51.71014,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.41678
            },
            {
              "py/object": "Point",
              "latitude": 51.69685,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.41132
            },
            {
              "py/object": "Point",
              "latitude": 51.69736,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.3968
            },
            {
              "py/object": "Point",
              "latitude": 51.69912,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.32555
            },
            {
              "py/object": "Point",
              "latitude": 51.69044,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.39387
            },
            {
              "py/object": "Point",
              "latitude": 51.69139,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.39656
            },
            {
              "py/object": "Point",
              "latitude": 51.68814,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.4139
            },
            {
              "py/object": "Point",
              "latitude": 51.67928,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.42126
            },
            {
              "py/object": "Point",
              "latitude": 51.67752,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.42307
            },
            {
              "py/object": "Point",
              "latitude": 51.66167,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.43328
            },
            {
              "py/object": "Point",
              "latitude": 51.66809,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.44435
            },
            {
              "py/object": "Point",
              "latitude": 51.68116547183463,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.4567277958160496
            }
          ],
          "subCodes": [
            "OX13 5BH",
            "OX13 5BJ",
            "OX13 5DT",
            "OX13 5DU",
            "OX13 5DX",
            "OX13 5DY",
            "OX13 5DZ",
            "OX13 5EA",
            "OX13 5EB",
            "OX13 5ED",
            "OX13 5EE",
            "OX13 5EF",
            "OX13 5EG",
            "OX13 5EH",
            "OX13 5EJ",
            "OX13 5EL",
            "OX13 5EN",
            "OX13 5EP",
            "OX13 5EQ",
            "OX13 5ER",
            "OX13 5ES",
            "OX13 5ET",
            "OX13 5EU",
            "OX13 5EW",
            "OX13 5FU",
            "OX13 5HG",
            "OX13 5HH",
            "OX13 5HJ",
            "OX13 5HL",
            "OX13 5HN",
            "OX13 5HP",
            "OX13 5HQ",
            "OX13 5HR",
            "OX13 5HS",
            "OX13 5HT",
            "OX13 5HW",
            "OX13 5HX",
            "OX13 5HY",
            "OX13 5HZ",
            "OX13 5JA",
            "OX13 5LS",
            "OX13 5LT",
            "OX13 5PL",
            "OX13 5QP",
            "OX13 6JP",
            "OX29 7QD"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM6402",
      "name": "OX13 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX13 5BH",
        "OX13 5BJ",
        "OX13 5DT",
        "OX13 5DU",
        "OX13 5DX",
        "OX13 5DY",
        "OX13 5DZ",
        "OX13 5EA",
        "OX13 5EB",
        "OX13 5ED",
        "OX13 5EE",
        "OX13 5EF",
        "OX13 5EG",
        "OX13 5EH",
        "OX13 5EJ",
        "OX13 5EL",
        "OX13 5EN",
        "OX13 5EP",
        "OX13 5EQ",
        "OX13 5ER",
        "OX13 5ES",
        "OX13 5ET",
        "OX13 5EU",
        "OX13 5EW",
        "OX13 5FU",
        "OX13 5GH",
        "OX13 5GN",
        "OX13 5HF",
        "OX13 5HG",
        "OX13 5HH",
        "OX13 5HJ",
        "OX13 5HL",
        "OX13 5HN",
        "OX13 5HP",
        "OX13 5HQ",
        "OX13 5HR",
        "OX13 5HS",
        "OX13 5HT",
        "OX13 5HW",
        "OX13 5HX",
        "OX13 5HY",
        "OX13 5HZ",
        "OX13 5JA",
        "OX13 5LS",
        "OX13 5LT",
        "OX13 5PL",
        "OX13 5QP",
        "OX13 6JP",
        "OX13 6PG",
        "OX29 7QD",
        "SN7 7QL",
        "SN7 7QQ",
        "SN7 8JE",
        "SN7 8PU",
        "SN7 8PX",
        "SN7 8PY",
        "SN7 8PZ",
        "SN7 8QA",
        "SN7 8QB",
        "SN7 8QD",
        "SN7 8QE",
        "SN7 8QF",
        "SN7 8QG",
        "SN7 8QH",
        "SN7 8QJ",
        "SN7 8QL",
        "SN7 8QN",
        "SN7 8QP",
        "SN7 8QQ",
        "SN7 8QR",
        "SN7 8QS",
        "SN7 8QT",
        "SN7 8QU",
        "SN7 8QW",
        "SN7 8QX",
        "SN7 8QY",
        "SN7 8QZ",
        "SN7 8RA",
        "SN7 8RB",
        "SN7 8RE",
        "SN7 8RG",
        "SN7 8RH",
        "SN7 8RJ",
        "SN7 8RL",
        "SN7 8RN",
        "SN7 8RP",
        "SN7 8RR",
        "SN7 8RS",
        "SN7 8RT",
        "SN7 8RU",
        "SN7 8RW",
        "SN7 8RX",
        "SN7 8RY",
        "SN7 8SA",
        "SN7 8SE",
        "SN7 8SF",
        "SN7 8SG",
        "SN7 8SQ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.4567277958160496,
              51.68116547183463
            ],
            [
              -1.44652,
              51.69227
            ],
            [
              -1.44225,
              51.69251
            ],
            [
              -1.43785,
              51.69266
            ],
            [
              -1.43614,
              51.69217
            ],
            [
              -1.43278,
              51.69141
            ],
            [
              -1.42472,
              51.69255
            ],
            [
              -1.41678,
              51.71014
            ],
            [
              -1.41132,
              51.69685
            ],
            [
              -1.3968,
              51.69736
            ],
            [
              -1.32555,
              51.69912
            ],
            [
              -1.39387,
              51.69044
            ],
            [
              -1.39656,
              51.69139
            ],
            [
              -1.4139,
              51.68814
            ],
            [
              -1.42126,
              51.67928
            ],
            [
              -1.42307,
              51.67752
            ],
            [
              -1.43328,
              51.66167
            ],
            [
              -1.44435,
              51.66809
            ],
            [
              -1.4567277958160496,
              51.68116547183463
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "2294be9f-be43-4116-b825-6bd759b99257",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4567277958160496,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-16T23:44:00.000+0000",
      "latitude": 56.53834150399983,
      "estimatedRestoration": "2022-02-17T14:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM5947",
      "reference": "JM5947",
      "name": "PH8 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PH8 0BX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -3.71362651748924,
          56.53834150399983
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-02-17T09:00:00.000+0000",
      "UUID": "2006622a-9f01-4766-adb8-ff6e207326f5",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -3.71362651748924,
      "resolved": false
    },
    {
      "customIncludeDefault": false,
      "py/object": "Fault",
      "loggedAt": "2022-02-17T12:43:00.000+0000",
      "latitude": 56.23090243263583,
      "estimatedRestoration": "2022-02-17T16:00:00.000+0000",
      "message": "We’re very sorry for the power cut. Our teams are working hard to restore your power, however, due to weather conditions that have impacted our network, and will continue to do so through to Friday, it may take longer than normal to restore power. We’re experiencing a busier than usual period and the conditions mean that even with additional teams we have mobilised, it’s much harder to repair faults due to the safety implications of working on our network in such conditions.<br/><br/>We will continue to update you with the latest information here on Power Track, so we’d recommend you register for updates on this fault or check back in later.<br/><br/>If you are in need of extra support, such as requiring electricity for medical equipment, you can call us on 105 where our team is on hand to help.<br/><br/>For updates on how we’re dealing with the weather, along with helpful advice, please <a href=\"https://www.ssen.co.uk/\" target=\"_blank\">click here<a/>.",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM6374",
      "reference": "JM6374",
      "name": "FK17 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "FK17 8HT"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -4.365595458333209,
          56.23090243263583
        ],
        "type": "Point"
      },
      "UUID": "2614c41b-7dd2-45a3-bf05-2123efbb7312",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.365595458333209,
      "resolved": false
    }
  ]
}
